contracts/contracts/strategies/BaseConvexMetaStrategy.sol

```diff
@@ -160,6 +165,27 @@ abstract contract BaseConvexMetaStrategy is BaseCurveStrategy {
         revert("Invalid Metapool asset");
     }
 
+    /**
+     * @dev Sets max withdrawal slippage that is considered when removing
+     * liquidity from Metapools.
+     *
+     * 1e18 == 100%, 1e16 == 1%
+     */
+    function setMaxWithdrawalSlippage(uint256 _maxWithdrawalSlippage)
+        external
+        onlyVaultOrGovernorOrStrategist
+    {
+        require(
+            _maxWithdrawalSlippage >= 1e15 && _maxWithdrawalSlippage <= 1e17,
```

Comment:
This is one of those two edged swords. In a depeg, the max slippage could go way below 10%, and we might want to pay that. On the other hand, it's giving the strategist a lot of power to go further than that.

Reply:
Hmm good point, in a serious de-peg this isn't very useful. What do you think the sweet spot is? maybe 30%?

For someone to take advantage of this they would need to compromise the strategist wallet and perform a flash loan attack.